contracts/v0.2/src/Feed.sol

```diff
@@ -6,34 +6,41 @@ import {IFeed} from "./interfaces/IFeed.sol";
 import {ITypeAndVersion} from "./interfaces/ITypeAndVersion.sol";
 
 contract Feed is Ownable, IFeed, ITypeAndVersion {
-    int256 private constant NOT_FOUND = -1;
-
     uint8 public override decimals;
     string public override description;
 
     uint64 private latestRoundId;
     mapping(uint64 => Round) internal rounds;
+
     address[] private oracles;
+    mapping(address => bool) private oracleWhitelist;
```

Comment:
how about naming as 'whitelist' instead of 'oracleWhitelist'? there aren't any other whitelist that could add a confusion at the moment

Reply:
Fixed in https://github.com/Bisonai/orakl/pull/1273/commits/94671483deaea0cabc5bfa0a250de1221455c7ce